HACKING.md

```diff
@@ -19,27 +19,37 @@ recommended to do that:
  - It avoids having to install development packages on your main machine.
  - It avoids having to map the build and test dependencies to package names of various distributions.
 
-On Fedora/CentOS/RHEL based distributions, install toolbox with
+1. Install `toolbox`
 
-    sudo dnf install toolbox
+   - Fedora/CentOS/RHEL based distributions:
 
-On Debian/Ubuntu based distributions, install it with
+         sudo dnf install toolbox
 
-    sudo apt install podman-toolbox
+   - Debian/Ubuntu based distributions:
 
-After that, create a cockpit development toolbox:
+         sudo apt install podman-toolbox
 
-    toolbox create --image quay.io/cockpit/tasks -c cockpit
+2. Create a development toolbox for Cockpit
 
-Enter the toolbox with
+       toolbox create --image quay.io/cockpit/tasks -c cockpit
 
-    toolbox enter cockpit
+3. Enter the toolbox:
+
+       toolbox enter cockpit
 
 Your home directory, user D-Bus, etc. are shared with the host, so you can
 edit files as you normally would. Building and running tests happens inside the
 toolbox container. If desired, you can install additional packages with
 `sudo dnf install`.
 
+The Cockpit team occasionally refreshes that container. To re-create yours from
+a new image, run
+```sh
+podman pull quay.io/cockpit/tasks
+toolbox rm cockpit
+```
+and then repeat steps 2 and 3 above.
```

Comment:
```suggestion
...and then repeat steps 2 and 3 from above.
```

It's a fragment that needs to be connected to the above, so ellipsis makes sense here.